packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,224 @@
+* [Runscope](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Runscope.html)
+* [Salesforce](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-in-Salesforce.html)
+* [Zoom.us](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Zoom.us.html)
+
+SCIM:
+
+* [Atlassian Cloud](https://confluence.atlassian.com/cloud/configure-user-provisioning-with-okta-957492956.html)
+* [Contentful](https://www.contentful.com/help/okta-user-provisioning-integration-with-scim/)
+* [Zscaler](https://help.zscaler.com/zia/saml-scim-configuration-guide-okta)
+
+### Configuration guide content
+
+The following are section suggestions for your configuration guide:
+
+* [Prerequisites](#prerequisites)
+* [Supported features](#supported-features)
+* [Procedure](#procedure)
+* [Troubleshooting tips](#troubleshooting-and-tips)
+
+#### Prerequisites
+
+In this section, specify any prerequisites required before your customer configures your integration in Okta. Examples may include enabling specific Okta features, enabling API access to your SCIM server, or adding a particular version of an integration in Okta.
+
+For example:
+
+When using SAML as the SSO mode with provisioning, you need to enable a specific account plan on the application side for silent activation.
+
+#### Supported features
+
+In this section of your guide, you want to outline what features your application supports. For example:
+
+* IdP-initiated SSO
+* SP-initiated SSO
+* SLO (Single Log Out)
+* Force Authentication
+* Create Users
+* Update User Attributes
+* Deactivate Users
+* Import Users
+* Import Groups
+* Sync Password
+* Profile Sourcing (formerly called Profile Mastering)
+
+Also include any restrictions. For example:
+
+Okta can't update user attributes for Admin users. This is an API limitation.
+
+> **Note:** You can briefly describe what each feature does. See the SCIM guides from the earlier example section.
+
+#### Procedure
+
+This section constitutes the majority of your guide and explains all the configuration steps needed to get your customers set up with your integration. Detail all settings and include any screenshots that can assist the user.
+
+Also include any best practices for your procedure, such as guidance on setting mappings for attributes, especially required attributes that don't have a default mapping. For example:
+
+> **Note:** The External ID is a required attribute, but it doesn't have a default mapping. This is because some customers prefer to set it to `EmployeeNumber`, and others like to set it to `emailAddress`. Assign the mapping to the correct value for your organization.
+
+#### Troubleshooting and tips
+
+You need to only include this section if there are known issues that apply to the entire configuration. In general, you should include best practices with the step-by-step procedure instructions.
+
+You may also want to include information on how to contact your organization if the customer has any support queries.
```

Comment:
Can we have a template for them to get started with? Ideally one, but could be one for each integration type if they are different enough.

Reply:
I was provided with this SCIM doc instruction template: https://saml-doc.okta.com/Provisioning_Docs/SCIM_Configuration_Guide_Instructions.pdf. @johnbaldo-okta , would you like to add this link as a template?